opentelemetry-api/runtime-attach/CustomLoggerPerClass/src/main/java/com/example/Main.java

```diff
@@ -0,0 +1,21 @@
+package com.example;
+
+import com.microsoft.applicationinsights.attach.ApplicationInsights;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class Main {
+
+  public static void main(String[] args) throws InterruptedException {
+    ApplicationInsights.attach();
+
+    // TODO (heya) fix duplicate log either by disabling log4j in Java agent
```

Comment:
I created [work item]( https://msazure.visualstudio.com/One/_workitems/edit/24459711) we discussed previously about changing the agent so auto-instrumented log capture can be turned off without affecting manual appender configuration


```
{
  "instrumentation": {
    "logging": {
      "level": "OFF"
    }
  }
}
```

Reply:
Another option: (could disable javaagent instrumentation for logback, log4j and jul)

* otel.instrumentation.logback-appender.enabled=false
* otel.instrumentation.log4j-appender.enabled=false
* otel.instrumentation.java-util-logging.enabled=false


```
{
  "instrumentation": {
    "logging": {
      "enabled": false
    }
  }
}
```

Reply:
https://github.com/microsoft/ApplicationInsights-Java/pull/3208